### PR TITLE
upload-555 integrate app insight

### PR DESCRIPTION
**Pull request by USZ7**
- Integrate App Insight SDK for tracing support.

## Reference
- https://learn.microsoft.com/en-us/azure/azure-monitor/app/worker-service

### CODE REVIEW

File: BulkFileUploadFunctionApp/BulkFileUploadFunction.cs [hunk 1]
```diff
@@ -94,6 +111,8 @@ public async Task Run([EventHubTrigger("%AzureEventHubName%", Connection = "Azur
 
                 await ProcessBlobCreatedEvent(blobCreatedEvent?.Data?.Url);
 
+                metric.TrackValue(1);
```
Comment (emmanuel-apau) [inline BulkFileUploadFunctionApp/BulkFileUploadFunction.cs:114]: what does this do?

Comment (USZ7) [reply BulkFileUploadFunctionApp/BulkFileUploadFunction.cs:114]: After processing each blob increment the count by 1 for each blob processed.

Comment (emmanuel-apau) [reply BulkFileUploadFunctionApp/BulkFileUploadFunction.cs:114]: ok could you send me a link via teams so i can see this an action

**Comment by emmanuel-apau** [comment]
Is there a wrapper that can be used vs the client specific commands?

**Comment by emmanuel-apau** [comment]
did you look at this page? https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-enable?tabs=aspnetcore#modify-your-application

**Comment by USZ7** [comment]
> Is there a wrapper that can be used vs the client specific commands?

I added wrapper class `TracingWrapper`

**Review by emmanuel-apau — APPROVED** [review]
lgtm. good work

**Pull request merged**